cmd/nvidia-ctk-installer/container/runtime/runtime.go

```diff
@@ -104,15 +111,20 @@ func Flags(opts *Options) []cli.Flag {
 	return flags
 }
 
-// ValidateOptions checks whether the specified options are valid
-func ValidateOptions(c *cli.Context, opts *Options, runtime string, toolkitRoot string, to *toolkit.Options) error {
+// Validate checks whether the specified options are valid
+func (opts *Options) Validate(logger logger.Interface, c *cli.Context, runtime string, toolkitRoot string, to *toolkit.Options) error {
 	// We set this option here to ensure that it is available in future calls.
 	opts.RuntimeDir = toolkitRoot
 
 	if !c.IsSet("enable-cdi-in-runtime") {
 		opts.EnableCDI = to.CDI.Enabled
 	}
 
+	if opts.ExecutablePath != "" && opts.RuntimeName != containerd.Name {
```

Comment:
I think we can add support for custom crio exec paths too

Reply:
I thought so too at first. The issue here is that for crio we have:
```
// CommandLineSource returns the CLI-based crio config loader
func CommandLineSource(hostRoot string) toml.Loader {
	return toml.LoadFirst(
		toml.FromCommandLine(chrootIfRequired(hostRoot, "crio", "status", "config")...),
		toml.FromCommandLine(chrootIfRequired(hostRoot, "crio-status", "config")...),
	)
}
```
Do you mean that we ONLY replace `crio` above and leave `crio-status` as is?